spinalcordtoolbox/scripts/sct_image.py

```diff
@@ -108,7 +118,8 @@ def get_parser():
     orientation = parser.add_argument_group('ORIENTATION OPERATIONS')
     orientation.add_argument(
         '-getorient',
-        help='Get orientation of the input image',
+        help='Get orientation of the input image. NOTE: This option has been deprecated, and will be removed in a '
```

Comment:
i have after thoughts: do we really want to remove this? few considerations:
- we will still have the API (in our code, we need to know what is the orientation)
- some users might be confused by the output of "sct_image -header" (people not used to the concept of qform/sform won't know how to get the orientation from there)
- it is not too much of a burden to maintain
- when we display the qform and sform, users don't know what SCT effectively uses (in case they are different). By selecting the one we rely on (ie: the qform), it will be clear for users what is the _effective_ orientation of the image (from SCT perspective)

Reply:
These are good points. Especially this one:

> some users might be confused by the output of "sct_image -header" (people not used to the concept of qform/sform won't know how to get the orientation from there)

I had been working under the assumption that our users knew how to parse `fslhd` output. This is really good to know, because it changes the warning message I was going to write for https://github.com/neuropoly/spinalcordtoolbox/issues/3249#issuecomment-803681048. 

Anyways, I'll undo the `-getorient` changes and take some time to investigate the cause of #3311 separately from this PR.

Reply:
> Anyways, I'll undo the -getorient changes and take some time to investigate the cause of #3311 separately from this PR.

maybe we can refactor the way we get the orientation, now that we parse the header, and just pick the first letter of the output qform orientation in each axis (ie: the way _we_ assess orientation from fslhd)